sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_builtin_evaluators.py

```diff
@@ -27,7 +27,7 @@
     SimilarityEvaluator,
     ViolenceEvaluator,
 )
-from azure.ai.evaluation.evaluators._eci._eci import ECIEvaluator
+from azure.ai.evaluation._evaluators._eci._eci import ECIEvaluator
```

Comment:
I understand this is for internal use, but why the namespace contains double "_eci"?

Reply:
I think it's because the folder and filename where the evaluator is available are both `_eci`

Reply:
@MilesHolland: is this evaluator for 1P customer? should we use this chance to fix this?

Reply:
Would be better change it to from azure.ai.evaluation._eci. import ECIEvaluator?

Reply:
We can probably do that in a separate PR. I'm waiting for this to go in, so I can fix the merge conflicts for the simulator.